### PR TITLE
Adding in python webserver

### PR DESCRIPTION
**Pull request by wpatterson-witekio**

### CODE REVIEW

File: recipes-apps/python-webserver/python-webserver.bb [hunk 1]
```diff
@@ -73,17 +78,18 @@ do_install() {
...
        install -m 0755 ${S}/html/usb3_results.html ${D}/home/root/webserver/html
        install -m 0755 ${S}/images/Avnet_logo_tagline_rgb.png ${D}/home/root/webserver/images
-       install -m 0755 ${S}/images/board.jpg ${D}/home/root/webserver/images
        install -m 0755 ${S}/images/favicon.ico ${D}/home/root/webserver/images
        install -m 0755 ${S}/pdfs/Delkin_Devices_Product_Line.pdf ${D}/home/root/webserver/pdfs
 }
 
 do_install_append_uz () {
        install -m 0755 ${S}/images/ultrazed.png ${D}/home/root/webserver/images
+       install -m 0755 ${S}/images/board.jpg ${D}/home/root/webserver/images
```
Comment (tnizan-witekio) [inline recipes-apps/python-webserver/python-webserver.bb:87]: so, you have to do this for 'pz'

File: recipes-apps/python-webserver/python-webserver.bb [hunk 2]
```diff
@@ -73,17 +78,18 @@ do_install() {
...
 }
 do_install_append_uz3eg-iocc () {
        install -m 0755 ${S}/pdfs/5043-PB-AES-ZU3EG-1-SOM-G-V3.pdf ${D}/home/root/webserver/pdfs
        install -m 0755 ${S}/pdfs/5080-PB-AES-ZU-IOCC-G-V2e.pdf ${D}/home/root/webserver/pdfs
+       install -m 0755 ${S}/images/board.jpg ${D}/home/root/webserver/images
```
Comment (tnizan-witekio) [inline recipes-apps/python-webserver/python-webserver.bb]: you don't have to add it here, as the 'uz3eg-iocc' is a 'uz' machine

File: recipes-apps/python-webserver/python-webserver.bb [hunk 3]
```diff
@@ -27,12 +27,12 @@ SRC_URI = "file://cgi.py \
      file://html/usb3_results.html \
      file://images/Avnet_logo_tagline_rgb.png \
      file://images/favicon.ico \
-     file://images/board.jpg \
```
Comment (tnizan-witekio) [inline recipes-apps/python-webserver/python-webserver.bb:30]: Could be easier to just call the minized image 'board.jpg' (even if it's not really the image of the board)

**Review by tnizan-witekio — CHANGES_REQUESTED** [review]

File: recipes-apps/python-webserver/python-webserver.bb (hunk 1 above)
Comment (wpatterson-witekio) [reply recipes-apps/python-webserver/python-webserver.bb:87]: Ah, yes. That is correct

File: recipes-apps/python-webserver/python-webserver.bb (hunk 2 above)
Comment (wpatterson-witekio) [reply recipes-apps/python-webserver/python-webserver.bb]: Good catch, I must've been jumping around too much. I intended to add this to pz for exactly that reason

**Review by tnizan-witekio — APPROVED** [review]

**Review by narrucmot — APPROVED** [review]

**Pull request merged**